README.md

```diff
@@ -12,8 +12,8 @@ This library contains reusable code for:
 ## Requirements
 
 - Boost 1.54
-- Ceres 1.12
-- Eigen 3.2.0
+- Ceres 1.13
+- Eigen 3.3
```

Comment:
in the ppa, you've got it listed as Eigen 3.3.2. Do you mind changing it here too, please? @leokoppel